### PR TITLE
Remove incorrect JavaDoc tags

### PR DESCRIPTION
**Pull request by thc202**
Remove incorrect JavaDoc tags, return tag when the method returns void
and param tag of inexistent type parameters.
Remove commented method and corresponding JavaDoc in BrowserDialog.

### CODE REVIEW

**Comment by thc202** [comment]
Done, commit and pull request description updated.

File: src/org/zaproxy/zap/session/SessionManagementMethodType.java [hunk 1]
```diff
@@ -37,8 +37,6 @@
  * The system automatically detects and loads {@link SessionManagementMethodType} classes and,
  * through them, the corresponding session management methods.
  * </p>
```
Comment (kingthorin) [inline src/org/zaproxy/zap/session/SessionManagementMethodType.java]: Paragraph marker is now un-needed?

Comment (thc202) [inline src/org/zaproxy/zap/session/SessionManagementMethodType.java]: Will remove it (it was not needed before either).

**Comment by thc202** [comment]
Change done.

**Pull request merged**